src/sendBankingTransaction.ts

```diff
@@ -14,10 +14,10 @@ export const sendBankingTransaction = (env:string,transactionData:object,hmacSig
             'Authorization': `hmac ${hashedPayload}`,
             'Content-Type': 'application/json'
         }
-        axios.post(`https://in.${env !== 'prod' ? env + '.' : '.' }killbills.${env !== 'prod' ? 'dev' : 'co'}/transaction`,JSON.stringify(transactionData),{headers}).then((response:any) => response?.data);
-        return true;
+        const sendTransaction = await axios.post(`https://in.${env !== 'prod' ? env + '.' : '.' }killbills.${env !== 'prod' ? 'dev' : 'co'}/transaction`,JSON.stringify(transactionData),{headers});
+        return await sendTransaction?.data;
```

Comment:
ca fait double await ?
```suggestion
        const sendTransaction = await axios.post(`https://in.${env !== 'prod' ? env + '.' : '.' }killbills.${env !== 'prod' ? 'dev' : 'co'}/transaction`,JSON.stringify(transactionData),{headers});
        return sendTransaction?.data;
```

Reply:
en effet inutile dans ce cas axios.post renvoie la promesse résolue